python/pyhail/dataset.py

```diff
@@ -908,9 +1306,92 @@ def join(self, right):
+
+        considers a model of the form
+
+        .. math::
+
+            \mathrm{height} = \beta_0 + \beta_1 \, \mathrm{gt} + \beta_2 \, \mathrm{age} + \beta_3 \, \mathrm{isFemale} + \beta_4 \, \mathrm{PC1} + \varepsilon, \quad \varepsilon \sim \mathrm{N}(0, \sigma^2)
+
+        where the genotype :math:`\mathrm{gt}` is coded as :math:`0` for HomRef, :math:`1` for
+        Het, and :math:`2` for HomVar, and the Boolean covariate :math:`\mathrm{isFemale}`
+        is coded as :math:`1` for true (female) and :math:`0` for false (male). The null
+        model sets :math:`\beta_1 = 0`.
+
+        Four variant annotations are then added with root ``va.linreg`` as shown
+        in the table. These annotations can then be accessed by other methods,
+        including exporting to TSV with other variant annotations.
+
+        +-------------------+------+-----------------------------+
+        |Annotation         |Type  |Value                        |
+        +-------------------+------+-----------------------------+
+        |``va.linreg.beta`` |Double|fit genotype                 |
+        |                   |      |coefficient,                 |
+        |                   |      |:math:`\hat\beta_1`          |
+        +-------------------+------+-----------------------------+
+        |``va.linreg.se``   |Double|estimated standard error,    |
+        |                   |      |:math:`\widehat{\mathrm{se}}`|
+        +-------------------+------+-----------------------------+
+        |``va.linreg.tstat``|Double|:math:`t`-statistic, equal to|
+        |                   |      |:math:`\hat\beta_1 /         |
+        |                   |      |\widehat{\mathrm{se}}`       |
+        +-------------------+------+-----------------------------+
+        |``va.linreg.pval`` |Double|:math:`p`-value              |
+        +-------------------+------+-----------------------------+
+
+        ``linreg`` skips variants that don't vary across the included samples,
+        such as when all genotypes are homozygous reference. One can further
+        restrict computation to those variants with at least :math:`k` observed
+        alternate alleles (AC) or alternate allele frequency (AF) at least
+        :math:`p` in the included samples using the options ``minac=k`` or
+        ``minaf=p``, respectively. Unlike the :py:meth:`.filter_variants_expr`
+        command, these filters do not remove variants from the underlying
+        variant dataset. Adding both filters is equivalent to applying the more
+        stringent of the two, as AF equals AC over twice the number of included
+        samples.
+
+        Phenotype and covariate sample annotations may also be specified using
+        `programmatic expressions <../reference.html#HailExpressionLanguage>`_
+        without identifiers, such as::
+
+            if (sa.isMale) sa.cov.age else (2 * sa.cov.age + 10)
```

Comment:
Put this in single quotes. Then it will look like a string -- same as rest of expressions in documentation.